test/integration_test/pdb_test.go

```diff
@@ -0,0 +1,160 @@
+package integrationtest
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"github.com/hashicorp/go-version"
+	corev1 "github.com/libopenstorage/operator/pkg/apis/core/v1"
+	"github.com/libopenstorage/operator/test/integration_test/types"
+	ci_utils "github.com/libopenstorage/operator/test/integration_test/utils"
+	"github.com/portworx/sched-ops/k8s/operator"
+
+	//operatorops "github.com/portworx/sched-ops/k8s/operator"
+	"github.com/sirupsen/logrus"
+
+	testutil "github.com/libopenstorage/operator/pkg/util/test"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	minSupportedK8sVersionForPdb, _ = version.NewVersion("1.21.0")
+)
+var testStorageClusterPDBCases = []types.TestCase{
+	{
+		TestName:        "PDBWithStoragelessNode",
+		TestrailCaseIDs: []string{"C257095"},
+		TestSpec: ci_utils.CreateStorageClusterTestSpecFunc(&corev1.StorageCluster{
+			ObjectMeta: metav1.ObjectMeta{Name: "test-stc"},
+		}),
+		ShouldSkip: func(tc *types.TestCase) bool {
+			if len(ci_utils.PxDeviceSpecs) == 0 {
+				logrus.Info("--portworx-device-specs is empty, cannot run PDBWithStoragelessNode test")
+				return true
+			}
+			kbVer, err := testutil.GetK8SVersion()
+			if err != nil {
+				logrus.Info("Skipping PDB test due to :", err)
+				return true
+			}
+			k8sVersion, _ := version.NewVersion(kbVer)
+			return ci_utils.PxOperatorVersion.LessThan(ci_utils.PxOperatorVer1_5_0) && k8sVersion.LessThan(minSupportedK8sVersionForPdb)
+		},
+		TestFunc: StoragelessNodePDB,
+	},
+	{
+		TestName:        "OverridePDBUsingValidAnnotation",
+		TestrailCaseIDs: []string{"C257148"},
+		TestSpec: ci_utils.CreateStorageClusterTestSpecFunc(&corev1.StorageCluster{
+			ObjectMeta: metav1.ObjectMeta{Name: "test-stc"},
+		}),
+		ShouldSkip: func(tc *types.TestCase) bool {
+			kbVer, err := testutil.GetK8SVersion()
+			if err != nil {
+				logrus.Info("Skipping PDB test due to :", err)
+				return true
+			}
+			k8sVersion, _ := version.NewVersion(kbVer)
+			return ci_utils.PxOperatorVersion.LessThan(ci_utils.PxOperatorVer23_10_2) && k8sVersion.LessThan(minSupportedK8sVersionForPdb)
+		},
+		TestFunc: OverridePDBUsingValidAnnotation,
+	},
+
+	{
+		TestName:        "OverridePDBUsingInvalidAnnotation",
+		TestrailCaseIDs: []string{"C257146", "C257203"},
+		TestSpec: ci_utils.CreateStorageClusterTestSpecFunc(&corev1.StorageCluster{
+			ObjectMeta: metav1.ObjectMeta{Name: "test-stc"},
+		}),
+		ShouldSkip: func(tc *types.TestCase) bool {
+			kbVer, err := testutil.GetK8SVersion()
+			if err != nil {
+				logrus.Info("Skipping PDB test due to :", err)
```

Comment:
`logrus.Infof("Skipping PDB test due to Err: %v", err)`

Reply:
We should come to some standard log and error prints, it will be easier to get your eyes trained on reading the log, but I think this is a better way to format the message